### ISSUE TITLE
Setup weekly zoom call for workgroup

### ISSUE THREAD

**Issue by prakesome** [pyatl/shovel#4]
Invite whoever would like to be a part of the efforts.
Meetings should be brief.
The point is to communicate whatever can't be done here.
Backlog grooming is definitely welcome
We dont need a paid zoom account or whatever. Just use whats free.
I guess we need some kind of calendar/invite.

**Comment by dcsim0n** [comment]
Here's a potential agenda for our first meeting. Talking about these topics will help us get next steps for the other items in the backlog.

PyATL meeting agenda

1. What is our mission? Why are we here?
    - Website mission statement?
2. What does “done” look like for meetup migration?
    - List of email addresses (probably)
    - What else?
3. Blog account creation
    - Who has credentials
    - What does “done” look like?
    - Is there a review process for blog posts? I.E. how do we create blog posts today?
4. Shared Calendar
    - Do we have the discord calendar plugin?
    - What are other options? (Google, etc)

**Comment by dcsim0n** [comment]
@prakesome @HasturDev, want to meet next week? Who else should be invited?

**Comment by prakesome** [comment]
Awesome, thanks Dana. Let’s figure out a day to meet every week until February 2024. No more than 15 mins unless required. Zoom or equivalent. Preferably during the evening. I prefer Tuesday, Wednesday, or Friday, but let’s get a consensus. Another point is we need a group calendar as well. But unclear how to proceed with that.  On Nov 24, 2023, at 3:47 PM, Dana Simmons ***@***.***> wrote:﻿
@prakesome @HasturDev, want to meet next week? Who else should be invited?

**Comment by HasturDev** [comment]
> Awesome, thanks Dana. Let’s figure out a day to meet every week until February 2024. No more than 15 mins unless required. Zoom or equivalent. Preferably during the evening. I prefer Tuesday, Wednesday, or Friday, but let’s get a consensus. Another point is we need a group calendar as well. But unclear how to proceed with that.  On Nov 24, 2023, at 3:47 PM, Dana Simmons ***@***.***> wrote:﻿ @prakesome @HasturDev, want to meet next week? Who else should be invited? —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

Yeah I'm down to just meet over discord every week because we can set it up on the calendar and put it in the admin chat

**Comment by dcsim0n** [comment]
> > Awesome, thanks Dana. Let’s figure out a day to meet every week until February 2024. No more than 15 mins unless required. Zoom or equivalent. Preferably during the evening. I prefer Tuesday, Wednesday, or Friday, but let’s get a consensus. Another point is we need a group calendar as well. But unclear how to proceed with that.  On Nov 24, 2023, at 3:47 PM, Dana Simmons _**@**_._**> wrote:﻿ @prakesome @HasturDev, want to meet next week? Who else should be invited? —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: _**@**_.**_>
> 
> Yeah I'm down to just meet over discord every week because we can set it up on the calendar and put it in the admin chat

That sounds like a good idea, and meets our Keep It Simple goal :).

Lets try meeting Wednesday. 

What time works well: Mornings, Lunch, or evenings after work?

**Comment by dcsim0n** [comment]
I added a bullet point to discuss a shared calendar.
